test/common.js

```diff
@@ -497,6 +497,12 @@ function fail(msg) {
 }
 exports.fail = fail;
 
+exports.mustNotCall = function(msg) {
+  return function dontCall() {
+    fail(msg || 'function should not have been called');
+  };
+};
+
```

Comment:
Just to add my 2 cents I also think common.mustNotCall() makes it immediately obvious what you want to happen without needing the context to know that 0 will imply it should not be called.